### DISCUSSION TITLE
01.%E5%90%8E%E7%AB%AF/06.Java/13.Java%E8%AE%A1%E6%97%B6

### DISCUSSION THREAD

**Discussion by giscus** [mirrormingzZ/mirrormingzZ.github.io#30] [Show and tell]
# 01.%E5%90%8E%E7%AB%AF/06.Java/13.Java%E8%AE%A1%E6%97%B6

A VitePress site

http://localhost:5173/01.%E5%90%8E%E7%AB%AF/06.Java/13.Java%E8%AE%A1%E6%97%B6.html

**Comment by pursue-wind** [comment]
# test1
## test2 
> qweqwe
### test3

tetwejqr
